code/scripting/api/objs/weapon.cpp

```diff
@@ -231,6 +231,10 @@ ADE_VIRTVAR(HomingPosition, l_Weapon, "vector", "Position that weapon will home
 		{
 			wp->homing_pos = vmd_zero_vector;
 		}
+		// need to update the position for multiplayer.
```

Comment:
More vertical white space would be greatly appreciated (in many of these changes, not just here) to help with readability. Particularly with `if` statement instances like this that can easily be mistaken for an if-else due to the lack of white space.